CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2587](https://github.com/Pycord-Development/pycord/pull/2587/))
 - Added optional `filter` parameter to `utils.basic_autocomplete()`.
   ([#2590](https://github.com/Pycord-Development/pycord/pull/2590))
+- Added `store_poll` method to `Webhook._WebhookState`.
+  ([#2624](https://github.com/Pycord-Development/pycord/pull/2624))
```

Comment:
This doesn't seem to match the nature of the PR. The changelog entry should be in the fixed section, instead.

Reply:
I wasn't sure which to do, given that it's fixing an issue by adding a new method